### PR TITLE
Reusable view tools

### PR DESCRIPTION
**Pull request by HebaMad**

### CODE REVIEW

File: UIDokan/Sources/UIDokan/Tools/CollectionviewHelper.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
+//
```
Comment (ahmdmhasn) [inline UIDokan/Sources/UIDokan/Tools/CollectionviewHelper.swift]: Let rename `CollectionViewHelper.swift` to `UICollectionView+DequeueHelper.swift` and move it to `Extensions`

File: UIDokan/Sources/UIDokan/Tools/TableViewHelper.swift [hunk 2]
```diff
@@ -0,0 +1,25 @@
+//
```
Comment (ahmdmhasn) [inline UIDokan/Sources/UIDokan/Tools/TableViewHelper.swift]: Let rename `UITableViewHelper.swift` to `UITableView+DequeueHelper.swift` and move it to `Extensions`

**Review by ahmdmhasn — COMMENTED** [review]
Great effort.. Just renaming 💪

**Review by ahmdmhasn — APPROVED** [review]

File: UIDokan/Sources/UIDokan/Tools/UICollectionView+DequeueHelper.swift [hunk 3]
```diff
@@ -0,0 +1,27 @@
+//
+//  File.swift
+//  
+//
+//  Created by heba isaa on 10/08/2022.
+//
+
+import Foundation
```
Comment (ziadali22) [inline UIDokan/Sources/UIDokan/Tools/UICollectionView+DequeueHelper.swift:8]: you can remove Foundation

**Review by ziadali22 — APPROVED** [review]

**Pull request merged**